_includes/sidenav.html

```diff
@@ -24,16 +23,17 @@
 
 <nav class="nav-secondary expandable {{ additional_classes }}">
     <h1 class="u-visually-hidden">
-    {%- for href, id, caption in items %}
+    {%- for item in items %}
```

Comment:
Well, the idea is that it's supposed to indicate for screenreader users that it's the navigation for that section, so it needs to apply to the menu as a whole, not just the current or parent page. Perhaps we need another variable for "navigation descriptor for screenreader users", which it will use if `index` is not set? And then a last ditch fallback of something generic like "section navigation"?